docs/community/hacktoberfest.md

```diff
@@ -0,0 +1,30 @@
+---
+id: hacktoberfest 
+title: 'Join us for Hacktoberfest! 🎃'
+sidebar_label: 'Join us for Hacktoberfest! 🎃'
+keywords:
+- hacktoberfest 
+---
+At OpenSauced, we love open source just as much as pizza! That's why we are participating in Hacktoberfest! 🎃
+
+## What is Hacktoberfest?
+
+ Hacktoberfest is an month-long challenge in October where people around the globe search for open source projects and make four pull requests. To learn more about this event, check out our blog post, [How to Participate in Hacktoberfest](https://dev.to/opensauced/how-to-participate-in-hackoberfest-13hm).
+
```

Comment:
Give instructions on how to join Hacktoberfest. This should include a link to sign-ups and basic instructions (4 PRs from Oct. 1-31 in repos opted into Hacktoberfest).